app/assets/stylesheets/ama_layout/layout_components/footer.scss

```diff
@@ -1,13 +1,37 @@
 // Sticky footer setup
-// See: https://philipwalton.github.io/solved-by-flexbox/demos/sticky-footer/
-.off-canvas-wrapper{
-  display: flex;
-  flex-direction: column;
-  min-height: 100vh;
+$footer-push-small: 617px;
+$footer-push-medium: 432px;
+$footer-push-large: 329px;
+
+html, body, .off-canvas-wrapper{
+  height: 100%;
 }
 
 .off-canvas-wrapper-inner{
-  flex: 1;
+  min-height: 100%;
+  @include breakpoint(small only){
+    margin-bottom: -$footer-push-small;
+  }
+  @include breakpoint(medium only){
+    margin-bottom: -$footer-push-medium;
+  }
+  @include breakpoint(large up){
+    margin-bottom: -$footer-push-large;
+  }
+
+  &:after{
+    content: "";
```

Comment:
We adapted the code from here: https://css-tricks.com/snippets/css/sticky-footer/.

If I remove the `content` property it breaks the footer, so I assume it's needed.